pkg/oc/cli/admin/release/new.go

```diff
@@ -46,7 +46,7 @@ func NewNewOptions(streams genericclioptions.IOStreams) *NewOptions {
 		// TODO: only cluster-version-operator and maybe CLI should be in this list,
 		//   the others should always be referenced by the cluster-bootstrap or
 		//   another operator.
```

Comment:
the signer server lives in the bootstrap node and dies with it I believe. How does this work it just makes sure the image is always available? The TODO seems to say other stuff shouldn't be here just trying to understand workflow.